registry/quilt_server/views.py

```diff
@@ -1079,12 +1080,14 @@ def access_put(owner, package_name, user):
                 "Invalid credentials"
                 )
         elif resp.status_code != requests.codes.ok:
+            print(resp.text)
             raise ApiException(requests.codes.server_error, "Server error")
         return dict()
 
     else:
         if user != PUBLIC:
-            resp = requests.get(OAUTH_PROFILE_API % user)
+            resp = requests.get(OAUTH_PROFILE_API % user,
```

Comment:
It just occurred to me that user probably needs to be url-encoded. I guess we restrict package names to only Python identifiers, but Django technically allows things like `+`, `@`, etc.

Reply:
True, but let's leave that for a different PR. This covers what's needed to get sharing working again and we've got an outstanding issue to restrict the allowed usernames to python-legal names anyway.